pkgs/tools/networking/wg-netmanager/default.nix

```diff
@@ -0,0 +1,40 @@
+# Test compilation/test with:
+#   nix build -f ~/src/nixpkgs wg-netmanager
+#
+# Test execution with:
+#   nix-env -f ~/src/nixpkgs -iA wg-netmanager
+
+{ lib, stdenv, fetchFromGitHub, rustPlatform, darwin }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wg-netmanager";
+  version = "0.2.8";
+
+  src = fetchFromGitHub {
+    owner = "gin66";
+    repo = "wg_netmanager";
+    rev = "wg_netmanager-v${version}";
+    sha256 = "16g5dfd4w0qx7pcgx6xw0py6w0s3dxwfb53j1xpsiz81412b71i4";
+  };
+
+  # related to the Cargo.lock file
+  cargoSha256 = "0f0qd6l3ra64rny2zsmsj4x3zrilqhnlp4l2l5sqglgkxg8g17jz";
+
+  buildInputs = lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Security;
+
+  # Testing this project requires sudo, Docker and network access, etc.
+  doCheck = true;
```

Comment:
The comment and code seem contradictory at first, maybe move the comment down to the checkFlags attribute?